diff --git a/setup.py b/setup.py
index a87c25e0..15f9ef15 100755
--- a/setup.py
+++ b/setup.py
@@ -881,19 +881,7 @@ if __name__ == '__main__':
                 CFLAGS.append('-msse2')
             blosc_sources += [f for f in glob.glob('c-blosc/blosc/*.c')
                               if 'sse2' in f]
-        # AVX2
-        if 'avx2' in cpu_flags:
-            print('AVX2 detected')
-            if os.name == 'nt':
-                if LooseVersion(platform.python_version()) >= LooseVersion('3.5.0'):
-                    # Neither MSVC2008 for Python 2.7 or MSVC2010 for Python 3.4 have
-                    # sufficient AVX2 support
-                    def_macros += [('__AVX2__', 1)]
-            elif compiler_has_flags(compiler, ["-mavx2"]):
-                CFLAGS.append('-DSHUFFLE_AVX2_ENABLED')
-                CFLAGS.append('-mavx2')
-            blosc_sources += [f for f in glob.glob('c-blosc/blosc/*.c')
-                              if 'avx2' in f]
+
     else:
         ADDLIBS += ['blosc']
 
